tutorial_plugin_java.md

```diff
@@ -0,0 +1,154 @@
+﻿---
+layout: page
+title: Tutorial Plugin Java
+permalink: /plugin/tutorial/java
+---
+
+# How to create a plugin for SmartSHARK
+This repository contains a minimal working example of a java plugin for SmartSHARK. This short readme is a tutorial on how to program a plugin starting from the ground. The first section will provide you with the basic information. More advanced information will be outlined for each file in the files section.
+
+## Basics
```

Comment:
This section is not needed, there are no other subsections. Just remove this and start directly with "## Get started"